tests/test_etl_measure.py

```diff
@@ -64,3 +78,20 @@ def test_update_measure_with_initial_data(mock_raw_query, mock_raw_query_data,
             sorted([(d[0], d[3], d[4]) for d in mock_raw_query_data], key=lambda d: d[0])
         }
     }
+
+
+@freeze_time('2017-07-01 13:00')
+@responses.activate
```

Comment:
I would a new [responses fixture](https://github.com/getsentry/pytest-responses/blob/b2e22923eb3814144733ce93f68b0d8506c0bf76/pytest_responses.py#L22-L25) and include it in the test dependencies when needed; that would save yousome typing and it feels more pytest-y 😺